webapp/tests/karma/ts/services/target-aggregates.service.spec.ts

```diff
@@ -84,22 +84,48 @@ describe('TargetAggregatesService', () => {
   });
 
   describe('isEnabled', () => {
-    it('should return false when user does not have permission', async () => {
-      authService.has.resolves(false);
+    it('should return true when user has permission and user has one facility assigned as array', async () => {
+      authService.has.resolves(true);
+      userSettingsService.get.resolves({ facility_id: [ 'facility-1' ] });
```

Comment:
I believe that an array with multiple elements would be better. Alternatively, if there is a specific reason for testing an array with only one element, please create a new test case using an array with multiple elements.

Reply:
> if there is a specific reason for testing an array with only one element

Must users (CHWs) will have only 1 element (facility) in this array, we are just including this case to play safe. 

> please create a new test case using an array with multiple elements.

There's one in L121 `it('should return false when user has more than one facility assigned',`
It should not return true when multiple elements are present; that's why we have this test case :)